archive/r/rust/README.md

```diff
@@ -7,6 +7,7 @@ Welcome to Sample Programs in Rust!
 - [Hello World in Rust](https://therenegadecoder.com/code/hello-world-in-rust/)
```

Comment:
We should probably start alphabetizing the sample programs in their respective READMEs.

Reply:
Done